scss/upload/_theme.scss

```diff
@@ -11,18 +11,35 @@
     $upload-success-bg: $success !default;
     $upload-disabled-text: darken($widget-bg, 27%) !default;
     $upload-disabled-border: darken($widget-bg, 27%) !default;
+    $upload-focused-status-box-shadow: 0 0 0 2px darken($input-bg, 6%) !default;
```

Comment:
This shadow is applied to the focused status icon only. Changing the var name to $upload-focused-box-shadow assumes that it will be applied to the Upload widhet itself. @gyoshev what do you think?